source/getting-started/installation-raspberry-pi-all-in-one.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: page
+title: "Raspberry Pi All-In-One Installer"
+date: 2016-05-12 01:39
+comments: true
+sharing: true
+footer: true
+---
+
+
+ 
+ Easily deploy a complete Home-Assistant server, complete with Websocket MQTT and Z-Wave support out of the box using Fabric! 
+ 
+ Start with a fresh Raspbian Jessie/Jessie-Lite or Debian 8 install, and the fabric script will do the following automatically:
+*  Create all needed directories
+*  Create needed service accounts
+*  Install OS and Python dependencies
+*  Setup a virtualenv to run Home-Assistant and components inside.
+*  Run as a service account
+*  Install Home-Assistant in a virtualenv
+*  Build and install Mosquitto from source with websocket support
+*  Build and Install Python-openzwave in the Home-Assistant virtualenv
+*  Add both Home-Assistant and Mosquitto to systemd services to start at boot
+
+<hr>
+
+**What is [Fabric](http://www.fabfile.org)?**
+ The official README says:
+>  "Fabric is a Python (2.5-2.7) library and command-line tool for streamlining the use of SSH for application deployment or systems administration tasks."
+ 
+<hr> 
+ 
+###  Usage:
+ **Setup:**
+ 
+ Simply install fabric locally:
+ ```pip3 install fabric3```
+ 
```

Comment:
Can we move the actual command to install to the top of the instructions and in an easy list. 

Also, did you test this page locally? I can already see some formatting issues with the way you wrapped code. Github markdown is not the same as for the site